src/SfxWeb/src/app/Models/DataModels/collections/CollectionBase.ts

```diff
@@ -81,6 +81,7 @@ export class DataModelCollectionBase<T extends IDataModel<any>> implements IData
                 return this.update(collection);
             }),
             catchError( err => {
+                this.collection = [];
```

Comment:
im a little hesitant on this. It makes sense for this situation to clear the collection but I am more so thinking in regards to previously how collections are used. I would think we should add a config to collectioBase thats like
clearOnFailureToUpdate and have it default to false and then check here to dictate whether or not to clear the collection.

Reply:
What I noticed was that after a failed request the collection wasn't affected, so the previous information stayed even though it shouldn't. I agree on what you commented, I'll test some things and be sure to add it in a new commit.